R/multiple-conditions.R

```diff
@@ -0,0 +1,182 @@
+#' @title Multiple chronic conditions
```

Comment:
In the title, can you identify what is different between the two multiple chronic disease questions?
Something like "Multiple chronic conditions (6 chronic conditions)"

I wondered about putting the names of the conditions in the title but that would be too long.

Reply:
I think "Multiple chronic conditions (6 chronic conditions)" would suffice in distinguishing between the two functions